mlir/lib/Dialect/plier_util/dialect.cpp

```diff
@@ -1041,6 +1041,57 @@ struct ChangeLayoutExpandShape
   }
 };
 
+struct ChangeLayoutSelect
```

Comment:
Please add comments, there is now way I can review without.

Reply:
done